### PR TITLE
feat(alert): add option for non-fixed alert (VIV-1315)

### PR DESCRIPTION
**Pull request by rachelbt**
Not sure about the property name...

### CODE REVIEW

File: libs/components/src/lib/alert/alert.scss [hunk 1]
```diff
@@ -51,6 +51,12 @@ $alert-transition-delay: --transition-delay;
 		margin-inline: 16px;
 	}
 
+	&:where(.in-flow) {
+		position: static;
+		margin: 16px;
```
Comment (RichardHelm) [inline libs/components/src/lib/alert/alert.scss:56]: Why does it have a built-in margin?

File: libs/components/src/lib/alert/README.md [hunk 2]
```diff
@@ -125,6 +125,17 @@ Use the `placement` attribute to set the location of the alert.
 <vwc-alert class="small-alert" placement="bottom-end" text="bottom-end" open></vwc-alert>
 ```
 
+### In-flow
```
Comment (RichardHelm) [inline libs/components/src/lib/alert/README.md:128]: An alternative could be to add a `position` attribute, with "fixed" being the default

Comment (rachelbt) [reply libs/components/src/lib/alert/README.md:128]: I thought about it - was afraid it was confusing with 'placement'...

File: libs/components/src/lib/alert/alert.scss (hunk 1 above)
Comment (rachelbt) [reply libs/components/src/lib/alert/alert.scss:56]: 👍  no reason. I'll remove it

File: libs/components/src/lib/alert/README.md (hunk 2 above)
Comment (rinaok) [reply libs/components/src/lib/alert/README.md:128]: We use an attribute called `strategy` in popup for this, and we use `fixed-dropdown` in select.

Comment (RichardHelm) [reply libs/components/src/lib/alert/README.md:128]: In the future we might want to add more options like `position: absolute`, so maybe `strategy` is good

**Review by RichardHelm — APPROVED** [review]

**Review by RichardHelm — APPROVED** [review]

**Pull request merged**